docs/source/index.html.md

```diff
@@ -17,82 +17,87 @@ includes:
 search: true
 ---
 
-# Introduction
+# General Working
+For a reputation system that records accross communities and which will be open to every community/team, it has to ensure that a user should not himself create a community and award themselves a lot of points. Also it has to make sure that score is transferable 
 
-Welcome to the Kittn API! You can use our API to access Kittn API endpoints, which can get information on various cats, kittens, and breeds in our database.
+i.e. 
+if ```A``` has a reputation score of ```100``` in a ```community of 100 people```   
```

Comment:
@omkarjc27 is it alright if we do this based on well defined levels instead points like we discussed on slack?

Reply:
The entire goal of the reputation system is to have a standardized way for developers to evaluate the skills/capabilities of the developers they work with. This reputation/level should be a measure of the true skill for an individual developer regardless of the community they collaborated in. 

The main reason for this is because the rep/level of a developer needs to be understood and shared across all communities that integrate it so there is no room for variance or independence in the way the reputation is granted. This goes back to the idea that reputation is not locked to a specific community. As developers move from community to community the capabilities of a "grand master" in one community should be roughly the same as the capabilities of a grand master in any other community - we need to maintain a standard for this to work. 

>Also it is difficult for us to decide on what basis the communities will grant points.

I don't see this being difficult because the idea was to emulate a system that already exists today inside engineering organizations. I'm not looking to invent something new but take something that already works and bring it to remote collaborative groups. [Example](http://www.reeoptical.com/images/IEEE_Engineering_Levels.pdf). This is an idea of well defined levels. 

>If a community comes up with a highly advanced new method of contributing to their projects with some brain waves or some absurd stuff, then it will become our responsibility to integrate that into our systems.
I'm not sure i follow your example - if this is true then they deserve to be at the level that describes their output and contributions to a project. Explicit tools used to accomplish the work are independent of the levels.

> Therefore, I highly suggest giving the communities all the freedom with respect to giving points/levels to their members,
> and I'm still persistent on my opinion that we should let the communities score/level their members/users as they want,
> then combine all the scores/levels that all the communities have given a user to form a universal rating or a universal level.

I hear what you are saying and this can work - it just feels like there is no reason to do any local/global calculation if a standard is set by us. We'd only be introducing complexity to end at the same result. 

From a developer perspective if we to choose points as the visible indication of capabilities/experience - we need to inform developers how to interpret the meaning of these points. How does a developer compare a person with 100 points vs 1000 point vs 10000. How big of a skill difference is this? This becomes a bit of a challenge - and if we allow for variance in how points are granted per community now we have to put another responsibility on community owner to tell developers how to approach the granting of points to each other within their community. Let's say we go with this approach now as developers move from community to community they have to understand a new set of rules for granting points - this is a barrier to adoption. 

My intention here not to take away independence but to provide a system that will be effective and easier to understand for all communities/developers.